Tests/test_file_palm.py

```diff
@@ -27,7 +27,7 @@ def open_with_magick(magick, tmp_path, f):
         magick + [f, outfile], stdout=subprocess.DEVNULL, stderr=subprocess.STDOUT
     )
     if rc:
-        raise OSError
+        assert False
```

Comment:
Would this work/be clearer?
```diff
-    if rc:
-        raise OSError
+    assert not rc
```

Reply:
Sure, that also works. I've made that change.